### PR TITLE
rules array and exclude-rule

### PR DESCRIPTION
**Pull request by eloymg**
Added posiblity of config multiple rulesets and exclude rules IDs

### CODE REVIEW

**Review by jesusfcr — CHANGES_REQUESTED** [review]
Just a detail

File: cmd/vulcan-semgrep/main.go [hunk 1]
```diff
@@ -77,8 +78,16 @@ func main() {
 			}
 		}
 		// Ensure not an empty ruleset is provided
-		if opt.Ruleset == "" {
-			opt.Ruleset = DefaultRuleset
+		var RulesetArray []string
+		switch opt.Ruleset.(type) {
+		case string:
+			RulesetArray = []string{fmt.Sprintf("%v", opt.Ruleset)}
+		case []interface{}:
```
[the remark this answers is not in the document]
Comment (eloymg) [reply cmd/vulcan-semgrep/main.go:85]: I put default value if array is empty

**Review by jesusfcr — APPROVED** [review]
Thanks for the contribution.

**Review by jesusfcr — APPROVED** [review]
LGTM

**Pull request merged**